### ISSUE TITLE
Python 3.10 Support

### ISSUE THREAD

**Issue by alanhdu** [conda-forge/pystan-feedstock#58]
### Comment:

Hello! It looks like Python 3.10 packages are not being built here (I only see 3.7, 3.8, and 3.9 packages), even though PyStan 3 does support 3.10.

Looking at the recipe, I'm not sure *why* 3.10 is not supported -- could it that we just need to rerender the repository (or merge in the PyStan 3.5 version bump)?

**Comment by rmax** [comment]
Indeed, looks like we need an update similar to https://github.com/conda-forge/lazrs-python-feedstock/commit/d22fab7a068782883db02ab6cb04e287fa640509

**Comment by alanhdu** [comment]
Ok -- I went ahead and asked for a rerender (and had to update the version of `httpstan`), but looks like this is working at https://github.com/conda-forge/pystan-feedstock/pull/59.

**Comment by rmax** [comment]
Thanks @alanhdu!
